README.md

```diff
@@ -161,6 +161,29 @@ CBPurchase.purchaseProduct(purchaseProductParams = purchaseProductParams, custom
  ```
 The above function will handle the purchase against Google Play Store and send the IAP token for server-side token verification to your Chargebee account. Use the Subscription ID returned by the above function, to check for Subscription status on Chargebee and confirm the access - granted or denied.
 
+### Upgrade or Downgrade Product
+Pass the `ChangeProductParams` and `CBCustomer` to the following function when the user chooses the product to purchase.
```

Comment:
Can we mention the function name rather than only saying "to the following function..."? It will make the sentence more clear.

Reply:
Pass the `ChangeProductParams` and `CBCustomer` to the following `ChangeProduct` function when the user chooses the product to purchase.